app/scss/graph.scss

```diff
@@ -0,0 +1,5 @@
+.show-graph {
+    height: 40vw;
+    width: 100%;
```

Comment:
Does this width do anything?

Reply:
Doesn't seem to - checked in the inspector and it looked like something was overriding it anyway. Removed it.